freqtrade/data/history.py

```diff
@@ -231,7 +235,9 @@ def download_pair_history(datadir: Optional[Path],
 
         misc.file_dump_json(filename, data)
         return True
-    except BaseException:
-        logger.info('Failed to download the pair: "%s", Interval: %s',
-                    pair, ticker_interval)
+
+    except Exception:
```

Comment:
I Love the explanation on why you changed BaseException to Exception! Great find btw!

Reply:
> I Love the explanation on why you changed BaseException to Exception! Great find btw!

Yeah, that's also for future devs. Now (after just an year) it's quite difficult to understand some changes made in, for example, backtesting by devs that are not active since those changes are not documented enough...